src/components/views/voip/VideoView.js

```diff
@@ -112,7 +112,9 @@ module.exports = React.createClass({
                  document.webkitFullscreenElement);
         const maxVideoHeight = fullscreenElement ? null : this.props.maxHeight;
         const localVideoFeedClasses = classNames("mx_VideoView_localVideoFeed",
-            {"mx_VideoView_localVideoFeed_flipped": UserSettingsStore.getSyncedSetting}
+            { "mx_VideoView_localVideoFeed_flipped":
+                UserSettingsStore.getSyncedSetting('VideoView.flipVideoHorizontally'),
```

Comment:
nitpick, but would be good to specify default false as the default default is null

Reply:
Fair!